runbotics/runbotics-scheduler/src/scheduler-database/tenant/tenant.entity.ts

```diff
@@ -38,4 +41,13 @@ export class Tenant {
 
     @OneToMany(() => ProcessContext, processContext => processContext.tenant)
     processContexts: ProcessContext[];
+
+    @OneToMany(() => CredentialAttribute, attribute => attribute.tenant)
+    attributes: CredentialAttribute[];
+
+    @OneToMany(() => Credential, credential => credential.tenant)
+    credentials: Credential[];
```

Comment:
Do we need it here? I think we can have only credentialCollections here, because attributes sits in credential and credentials sits in collection?